ios/Reader/EPUB/EPUBViewController.swift

```diff
@@ -10,10 +10,14 @@ class EPUBViewController: ReaderViewController {
       locator: Locator?,
       bookId: String
     ) throws {
+      let httpClient = DefaultHTTPClient()
```

Comment:
I'm not sure we should build a new HTTPServer on every init of an `EPUBViewController`. I think perhaps we should have a singleton instance of it or something.